django/db/backends/base/creation.py

```diff
@@ -47,6 +67,8 @@ def create_test_db(self, verbosity=1, autoclobber=False, serialize=True, keepdb=
                 self._get_database_display_str(verbosity, test_database_name),
             ))
 
+        self.mark_tests()
```

Comment:
This should only be called when the Django test suite is running. Perhaps with an environment variable which `runtests.py` could set, unless we can come up with a more elegant solution.